moveit_core/constraint_samplers/test/pr2_arm_kinematics_plugin.cpp

```diff
@@ -188,17 +192,18 @@ int PR2ArmIKSolver::cartToJntSearch(const KDL::JntArray& q_in, const KDL::Frame&
       return -1;
     q_init(free_angle_) = initial_guess + search_discretization_angle_ * count;
     if (verbose)
-      ROS_DEBUG_NAMED("pr2_arm_kinematics_plugin", "%d, %f", count, q_init(free_angle_));
-    loop_time = (ros::Time::now() - start_time).toSec();
+      RCLCPP_WARN(LOGGER, "%d, %f", count, q_init(free_angle_));
+    time = rclcpp::Clock().now();
+    loop_time = time.seconds() - start_time.seconds();
```

Comment:
```suggestion
    loop_time = (rclcpp::Clock().now() - start_time).seconds();
```
Then we don't need to introduce `time`.

Reply:
https://github.com/ros-planning/moveit2/pull/60/commits/a527755c9757ef9a9aa3902ceb45adc84fc9ceb7